bitnami/contour/templates/default-backend/ingress.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
     {{- end }}
   annotations:
-    kubernetes.io/ingress.class: {{ include "contour.ingressClassName" . }}
+    kubernetes.io/ingress.class: {{ default "contour" (include "contour.ingressClassName" .) }}
```

Comment:
Shouldn't the default value come from `include "contour.ingressClassName"`?

Reply:
I reverted this after applying https://github.com/bitnami/charts/pull/8046#discussion_r744552904 🙂